### ISSUE TITLE
bckenstler/CLR

### ISSUE THREAD

**Issue by shinerio** [shinerio/reading#20]
<b>bckenstler/CLR</b><br>
This repository includes a Keras callback to be used in training that allows implementation of cyclical learning rate policies, as detailed in Leslie Smith's&hellip;<br>
<br>
August 16, 2018 at 07:48PM<br>
via Instapaper https://ift.tt/2ntybDC
